features/bevragen/persoon/verblijfplaats/dev/onbekendwaardes-gba.feature

```diff
@@ -0,0 +1,120 @@
+# language: nl
+
+
+Functionaliteit: properties die gevuld zijn met standaard-waardes worden ongewijzigd geleverd voor de verblijfplaats. 
+
+  Wanneer in de registratie specifieke waarden gereserveerd zijn voor een onbekende waarde, worden deze waarden ongewijzigd geleverd door de GBA-API.
+  Wanneer een element in de registratie een standaardwaarde heeft, die betekent dat de waarde onbekend is, wordt het corresponderende veld opgenomen in de response met de betreffende waarde. 
+
+
+Rule: de volgende standaardwaarden worden ongewijzigd geleverd bij de verblijfplaats
+  Het gaat om de volgende properties en standaardwaardes van de persoon:
+  | property                                         | standaardwaarde  |
+  | ------------------------------------------------ | ---------------- |
+  | verblijfplaats.straat                            | .                |
+  | verblijfplaats.huisnummer                        | 0                |
+  | verblijfplaats.woonplaats                        | .                |
+  | verblijfplaats.nummeraanduidingIdentificatie     | 0000000000000000 |
+  | verblijfplaats.adresseerbaarObjectIdentificatie  | 0000000000000000 |
```

Comment:
land adres buitenland mist in dit lijstje

Reply:
Toegevoegd